src/Employer/Employer.Web/Views/Part2/ApplicationProcess/ApplicationProcess.cshtml

```diff
@@ -18,21 +18,21 @@
                 <div esfa-validation-marker-for="ApplicationMethod" class="form-group">
                     <fieldset>
                         <label class="form-label-bold sfa-small-bottom-margin" id="@Anchors.ApplicationMethodHeading">
-                            Application process?
-                            <span class="form-hint">How will candidates apply for this vacancy</span>
+                            How will candidates apply for this vacancy?
+                            <span class="form-hint">All vacancies will be published on Find an Apprenticeship</span>
```

Comment:
Elsewhere and lower on this page we have `Find an apprenticeship` which I have spoken with Karen about and we need to keep the word `apprenticeship` lowercase and only the `F` in `Find` uppercase.